doozer/doozerlib/cli/release_gen_assembly.py

```diff
@@ -517,6 +546,8 @@ def _generate_assembly_definition(self) -> dict:
 
         if self.assembly_type not in [AssemblyTypes.CUSTOM, AssemblyTypes.PREVIEW]:
             group_info['advisories'], group_info["release_jira"] = self._get_advisories_release_jira()
+            if group_info['advisories']['prerelease'] == -1:
+                group_info['operator_index_mode'] = 'pre-release'
```

Comment:
should this be outdented? I think we'll only need this when `assembly_type` is `PREVIEW`

also `['prerelease']` won't always be defined so shouldn't it be `.get('prerelease')`?

Reply:
Ah good catch, I've updated the block to not run only for custom assembly